api/src/main/resources/messages.properties

```diff
@@ -3292,3 +3292,6 @@ OrderFrequency.concept.shouldBeClassFrequency=Concept should be of class Frequen
 OrderFrequency.concept.shouldNotBeShared=Only one OrderFrequency should be mapped to a particular concept
 
 Module.error.cyclicDependencies=Could not start some modules because they depend on others which are not started. Please first start them or use the "Start All" button on the "Manage Modules" page: {0}
+
+Condition.conditionShouldNotBeNull=Condition is required for the Condition object
```

Comment:
Remove "the Condition object" from these messages

Reply:
Done